### PR TITLE
Timedated logging and service priority

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/timedate/timedated.c [hunk 1]
```diff
@@ -486,8 +490,12 @@ static int unit_enable_or_disable(UnitStatusInfo *u, sd_bus *bus, sd_bus_error *
 
         /* Call context_update_ntp_status() to update UnitStatusInfo before calling this. */
 
-        if (streq(u->unit_file_state, "enabled") == enable)
+        if (streq(u->unit_file_state, "enabled") == enable) {
+                log_unit_debug(u, "is already %s.", enable_disable(enable));
```
Comment (yuwata) [inline src/timedate/timedated.c]: `%sd`.

(I prefer to drop 'is', but ok)

**Comment by keszybz** [comment]
Updated.

**Comment by yuwata** [comment]
Hmm... I thought that the third commit makes `timedatectl set-ntp on` always enable and start the service in highest priority, and disable and stop the other services. But it does not. Is it intended?
```
$ cat /usr/lib/systemd/ntp-units.d/90-hoge.list 
hoge.service
$ systemctl cat hoge.service
# /etc/systemd/system/hoge.service
[Service]
Type=simple
RemainAfterExit=yes
ExecStart=sleep 1h
$ sudo systemctl start hoge.service
$ sudo timedatectl set-ntp on
$ systemctl is-active hoge.service
active
```

**Comment by keszybz** [comment]
I don't know what other .list files you have, but I don't see anything wrong with this transcript. Can you do `systemd-analyze cat-config systemd/ntp-units.d` ?

**Comment by yuwata** [comment]
I have the default one, i.e., the file for systemd-timesyncd.service. I will show the result of the command later.

**Comment by yuwata** [comment]
@keszybz I've updated the above comment.

**Comment by keszybz** [comment]
I'm an idiot. Should be OK now.

**Comment by yuwata** [comment]
LGTM, and works fine for me.

**Pull request merged**